spec/unit/type/file/content_spec.rb

```diff
@@ -227,6 +227,30 @@
           end
         end
       end
+
+      SAVED_TIME = Time.now
+      [:ctime, :mtime].each do |time_stat|
+        [["older", SAVED_TIME-1, false], ["same", SAVED_TIME, true], ["newer", SAVED_TIME+1, true]].each do
+          |compare, target_time, success|
```

Comment:
Also wondering if we need to account for ctime/mtime for directories and symlinks? Do they have the same meaning as files?